config/config/common-config.yaml

```
@@ -815,6 +815,12 @@ opa:
     tags:
       - latest
 
+  ## Prevent the creation of load balancer services when unsupported
+  rejectLoadBalancerService:
+    enforcement: deny
+    enabled: false
```

Nit-pick: all of the others have it this order
```suggestion
  rejectLoadBalancerService:
    enabled: false
    enforcement: deny
```

---

I also wonder if we should change the value of "enabled" based on the infra provider (because that is usually what dictates if we can use loadbalancer services or not), by adding some code in the init steps (similar to [this](https://github.com/elastisys/compliantkubernetes-apps/blob/main/bin/init.bash#L197)). What do you think about adding that kind of thing? You should be able to copy the loadbalancer values in the function I linked.